sympy/matrices/reductions.py

```diff
@@ -313,6 +313,14 @@ def _rref(M, iszerofunc=_iszero, simplify=False, pivots=True,
     [0, 1]])
     >>> rref_pivots
     (0, 1)
+    >>> m.rref(pivots=False)
+    Matrix([
+    [1, 0],
+    [0, 1]])
+    >>> m.rref(pivots=True)
+    (Matrix([
+    [1, 0],
+    [0, 1]]), (0, 1))

```

Comment:
Since True is the default I don't think we need this example. It would be better just to clarify that True is the default in the parameters section.

Reply:
ohhk I will remove it